### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by nokome** [stencila/stencila#797] [General]
## 👋 Welcome!

This is a place for members of our community to connect. Please:

  * 🤔 Ask questions you’re wondering about.
  * 💡 Share your ideas for new Stencila features.
  * 📢  Make announcements that other members will be interested in.
  * 💬 Kick start discussions on things of mutual interest.
  * 🤝 Engage with other community members.
  * 👋 Welcome others and be open-minded.
  *  💪 Remember that this is a community we build together.

To get started, comment below with an introduction to yourself and how you are involved in the community.

**Comment by finlay** [comment]
Hi Nokome

I think I like this venue. Lets see if we can get a conversation going here.



Finlay

**Reply by nokome** [reply]
Thanks @finlay, @fkmendes and I will create a new discussion re LaTeX / Rnw
